client/src/Components/FirstSection.css

```diff
@@ -0,0 +1,30 @@
+#marginRow {
+    margin-top: 5vh;
+    margin-bottom: 5vh;
+    font-family: roboto;
+}
+
+.img0Div{
+    height: 500px;
+    width: auto;
+    aspect-ratio: 1;
+    margin: 0 auto;
+    background-position: center center;
+}
+.img0 {
+    border-radius: 50%;
+    background-image: url('../Assets/img1.jpg');
+    background-size: cover;
+    height: 100%;
+    width: 100%;
+    background-position: center center;
+   
+    
+}
+
+
+.cardAlign {
+position: relative;
+  top: 30%;
+
+}
```

Comment:
Just a couple things I'm noticing here.

1. This very closely resembles the class from `Cards.css`. If this is a style that cards should be using, it might be more appropriate to add it to the `Cards.css`.
2. There doesn't seem to be any indentation on line 27.
3. Line 28 looks like 2 spaces of indentation (although the rest of the file looks to be using 4 spaces).
4. There's an empty line on line 29.
5. There's no new line at the end of the file.